### ISSUE TITLE
Allow window resizing

### ISSUE THREAD

**Issue by Silvia-Dragoness** [NicolasConstant/msfs-community-downloader#70]
Please unlock the window size and allow resizing.  It's rather frustrating having such limited window space.

**Comment by NicolasConstant** [comment]
But windows size is unlocked? You can resize by dragging any border.

**Comment by Silvia-Dragoness** [comment]
The "maximize" window button is disabled.

**Comment by NicolasConstant** [comment]
Yes I have a bug with it and I had to remove it, I'll check again if I can fix it.

**Comment by r-flo-r** [comment]
would also vote for that "issue". grabbing the border to resize is possible but not conventient, than rather having the maximize button (and then windows may remembers the window size on the next startup and you finally get rid of it completely)
thanks
